pages/release-notes.mdx

```diff
@@ -139,6 +139,124 @@ Memgraph (v2.11.1 and lower).
 - A bug introduced in v2.12.0 with failed key encryption when
   adding OpenAI and Azure OpenAI keys is fixed now.
 
+## Memgraph v2.16.0 - Apr 10, 2024
+
+{<h3> New features and improvements </h3>}
+
+- Added a new command, [`DROP GRAPH`](/querying/clauses/drop-graph), that
+  deletes data in the database in an efficient manner.
+  [#1723](https://github.com/memgraph/memgraph/pull/1723)
+- Fixed a bug that caused the incorrect coordinator to become the leader due to
+  missing cluster state data.
+  [#1897](https://github.com/memgraph/memgraph/pull/1897)
+- Added [automatic index creation](/fundamentals/indexes) for node labels and
+  edge types. Two new configuration flags control this feature for labels and
+  edge-types automatically encountered by the database.
+  [#1841](https://github.com/memgraph/memgraph/pull/1841)
+- Added support for `bolt+routing` when connecting to a cluster of Memgraph
+  instances. [#1796](https://github.com/memgraph/memgraph/pull/1796)
+- Added support for [edge type indices](/fundamentals/indexes#edge-type-index)
```

Comment:
I would add this above automatic index creation, so users first see what's new and how that's connected to the automatic index creation